anitya/lib/backends/github.py

```diff
@@ -88,8 +88,8 @@ def get_versions(cls, project):
             (owner, repo) = url.split('/')
         except ValueError:
             raise AnityaPluginException(
-                'Project %s was incorrectly set-up. \
-                Can\'t parse owner and repo' % project.name)
+                'Project %s was incorrectly set-up. '  % project.name +
+                'Can\'t parse owner and repo.')
```

Comment:
It's neater to do it as:

``"Project {} was incorrectly set-up. Can't parse owner and repo.".format(project.name)``

rather than having the format in the middle of a string